amgut/lib/data_access/ag_data_access.py

```diff
@@ -412,6 +412,11 @@ def deleteAGParticipant(self, ag_login_id, participant_name):
                    "survey_id = %s")
             curr.execute(sql, [survey_id])
 
+            # Reset survey attached to barcode(s)
+            sql = ("UPDATE ag_kit_barcodes SET survey_id = NULL WHERE "
```

Comment:
I know the tests are passing but I'm not sure if this will work in the live site, this based as in the DB survey_id is a not nullable FK, right? Perhaps I'm looking at this wrongly.